Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM golang:1.9.3
+
+RUN go get -u github.com/jstemmer/go-junit-report
+RUN go get -u github.com/smartystreets/goconvey
+RUN go get -u golang.org/x/tools/cmd/cover
+RUN go get -u github.com/axw/gocov/...
+RUN go get -u github.com/AlekSi/gocov-xml
+RUN go get -u golang.org/x/tools/cmd/goimports
+RUN go get -u github.com/golang/lint/golint
```

Comment:
what about using go dep instead? so that both docker and non-docker users have a standard way to install deps.

Reply:
@bamarni we could do that, possibly in another PR.